scripts/objectscale-install-main.sh

```diff
@@ -15,27 +15,31 @@ function usage()
 {
    cat << HEREDOC
 
-   Usage: $progname  --set type=<install_type> --set helmrepo=<repo> --set primaryStorageClassName=... [--set global.registry=myfavereg.docker.com]
-                        [--verbose] [--debug]
+   Usage: $progname  --set type=<install_type> --set helmrepo=<repo> --set primaryStorageClassName=... [--set global.registry=myfavereg.docker.com] [--set global.watchAllNamespaces=false]
+                        [--verbose] [--debug] [--namespace default]
 
-   required arguments:
+   required arguments for install:
      --set type={install|list|remove}       set install type
      --set helmrepo=repo_name               location of helm repository or directory where charts located
-     --set primaryStorageClassName=<string> name of storage class to install high performance persistent volumes (e.g. csi-baremetal-ssdlvg)
+     --set primaryStorageClassName=<string> name of storage class to install high performance persistent volumes (e.g. csi-baremetal-ssdlvg). Required only for type=install
 
    optional arguments:
      --set global.registry=<string>           where to pull ObjectScale container images
      --set global.registrySecret=<string>     what registry login to use to pull down container images
      --set secondaryStorageClassName=<string> name of storage class for normal performance persistent volumes (e.g. csi-baremetal-hddlvg)
+     --set global.watchAllNamespaces=<true|false> allow deploying objectstore across the namespaces
      -h, --help                               show this help message and exit
      -v, --verbose                            increase the verbosity of the bash script
      --debug                                  shell debug (-x) mode, lots of output
+     --namespaace                             namespace to install the plugin
 
    Example for OpenShift:
    ----------------------
    helm repo add objectscale "https://MY_PRIVATE_TOKEN@raw.githubusercontent.com/emcecs/charts/v0.7x.0/docs"
    helm repo update
    ./objectscale-install.sh --set type=install --set helmrepo=objectscale --set global.registry=asdrepo.isus.emc.com:8099 --set primaryStorageClassName=csi-baremetal-sc-ssdlvg --set secondaryStorageClassName=csi-baremetal-sc-hddlvg
+   helm repo add objectscale "https://MY_PRIVATE_TOKEN@raw.githubusercontent.com/emcecs/charts/v0.7y.0/docs"
```

Comment:
```suggestion
   helm repo add objectscale "https://MY_PRIVATE_TOKEN@raw.githubusercontent.com/emcecs/charts/v0.yy.0/docs"
```
only a suggestion so you don't have to update this again in 2 sprints